app/model/Layer.js

```diff
@@ -16,7 +16,7 @@ Ext.define('MoMo.admin.model.Layer', {
         name: 'name',
         type: 'string'
     }, {
-        name: 'type',
+        name: 'hoverFieldTemplate',
```

Comment:
yes template has moved to appearance. and the layer should have a field `dataType`.
Maybe you can check if upload for raster and vector is still working after merge, becuase thats related to that